client/src/driver/BitBakeProjectScanner.ts

```diff
@@ -188,6 +191,10 @@ export class BitBakeProjectScanner {
     this._bitbakeScanResult._includes = this.searchFiles(this._includeFileExtension)
   }
 
+  private scanForConfFiles (): void {
+    this._bitbakeScanResult._confFiles = this.searchFiles(this._confFileExtension)
```

Comment:
some conf files may also be present in the build directory (build/conf/local.conf). You can get the build directory only through asking bitbake similarly to the overrides scan in this file. If you need this, I suggest refactoring the override scan and this one to use a global `bitbake -e` and get the required variables values.

Reply:
This is expected because bitbake -e will only show the conf files that are actually used by the recipe. It is more accurate.

Reply:
In this case, I will get the overrides and conf files through the command. I should get some inspiration from how we get the recipes.

Reply:
Actually, this might cause some regression if we use the conf files from the `bitbake -e`. See the example below:
![image](https://github.com/yoctoproject/vscode-bitbake/assets/47988425/51df6be9-5a68-48e2-a04e-9c83f2abfa07)
The recipe is `meta/recipes-core/systemd/systemd-boot_254.4.bb`. The line with highlight should point to `meta/conf/image-uefi.conf` if not mistaken.
However, the output from `bitbake -e` has nothing related to `image-uefi.conf` and the output from `bitbake -e systemd-boot` doesn't have it in the `INCLUDE HISTORY` section. Thought the `image-uefi.conf` appears in the operations above some variables. This is the same issue that I mentioned in another conversation: Missing a clear pattern.

Reply:
Nice catch! didn't expect that from bitbake -e.